front-project/src/features-store/auth/auth.reducer.js

```diff
@@ -1,13 +1,16 @@
-import { TOGGLE_IS_LOGGED_IN } from "./auth.constants";
+import { AUTH_SIGN_IN, AUTH_SIGN_UP } from "./auth.constants";
 
 const initialState = {
-  isLoggedIn: true,
+  user: null,
 };
 
 export const authReducer = (state = initialState, action) => {
   switch (action.type) {
-    case TOGGLE_IS_LOGGED_IN:
-      return { ...state, isLoggedIn: !state.isLoggedIn };
+    case AUTH_SIGN_IN:
```

Comment:
it's the same actions. maybe create action AUTH_SET_USER and use it in both thunks?